<dec f='boost-1.85.0/libs/contract/include/boost/contract/core/exception.hpp' l='376' type='const boost::contract::from_failure_handler &amp; boost::contract::exception_::set_entry_inv_failure_unlocked(const boost::contract::from_failure_handler &amp; f)'/>
<def f='boost-1.85.0/libs/contract/include/boost/contract/detail/inlined/core/exception.hpp' l='395' ll='399' type='const boost::contract::from_failure_handler &amp; boost::contract::exception_::set_entry_inv_failure_unlocked(const boost::contract::from_failure_handler &amp; f)'/>
<use f='boost-1.85.0/libs/contract/include/boost/contract/detail/inlined/core/exception.hpp' l='405' u='c' c='_ZN5boost8contract10exception_28set_entry_inv_failure_lockedERKNS_8functionIFvNS0_4fromEEEE'/>
